storybook/manager.tsx

```diff
@@ -3,9 +3,18 @@
 import React from "react"
 import { addons } from "@storybook/addons"
 import { defaultTheme } from "@kaizen/design-tokens"
-import { CATEGORIES_ICON } from "./constants"
 import KaizenTheme from "./theme"
 
+const CATEGORIES_ICON = {
+  Introduction: "👋",
+  Components: "⚙️",
+  Helpers: "🤝",
+  "Design Tokens": "🎨",
+  Deprecated: "💣",
+  Systems: "🤖",
+  AIO: "📦",
+}
+
 const colors = defaultTheme.color
 
 addons.setConfig({
```

Comment:
GTM still exists here?

Reply:
This came through with the new theming stuff 😬
To delete.